### PR TITLE
Fixes #9157: Allows customizing the crane port.

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ekohl** [comment]
:+1:

**Comment by jlsherrill** [comment]
for some reason the port isn't showing up in my crane.conf:

```
[general]
debug: false
#data_dir_polling_interval: 60
data_dir: /var/www/pub/docker/app
endpoint: katello-centos7.example.com:
```

**Comment by ehelms** [comment]
@jlsherrill turns out it was a typo, updated with a test

**Comment by ehelms** [comment]
@jlsherrill updated

**Comment by ehelms** [comment]
@jlsherrill Rebased, if you don't think this change is useful, please let me know and I will close both related PRs.

File: .fixtures.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 fixtures:
   repositories:
-    stdlib:        "git://github.com/puppetlabs/puppetlabs-stdlib.git"
     apache:        "git://github.com/puppetlabs/puppetlabs-apache.git"
-    foreman:       "git://github.com/theforeman/puppet-foreman.git"
+    stdlib:        "git://github.com/puppetlabs/puppetlabs-stdlib.git"
     concat:        "git://github.com/puppetlabs/puppetlabs-concat"
-    certs:         "git://github.com/katello/puppet-certs.git"
-    common:        "git://github.com/katello/puppet-common.git"
+    file_concat:
```
Comment (ekohl) [inline .fixtures.yml]: I don't think this change is needed anymore.

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**